src/main/java/com/core/Response.java

```diff
@@ -0,0 +1,207 @@
+package com.core;
+
+import com.core.savedata.SaveFile;
+import com.util.json.JsonArray;
+import com.util.json.ReadWriteFiles;
+import com.util.datetime.DateTime;
+import java.util.stream.IntStream;
+
+import com.util.Printer;
+import com.tasks.Deadline;
+import com.tasks.DoableTask;
+import com.tasks.Event;
+import com.tasks.Todo;
+import java.util.stream.Stream;
+
+public enum Response {
```

Comment:
Maybe it may be better to split up the responses into command and error classes, so that in the future if more functionalities get added, it reduces the code from getting too bloated.